src-ui/src/app/data/paperless-sso-group.ts

```diff
@@ -0,0 +1,6 @@
+import { ObjectWithId } from './object-with-id'
+
+export interface PaperlessSSOGroup extends ObjectWithId {
+  name?: string
+  group?: number
```

Comment:
```suggestion
  name: string
  group: number
```

I assume these aren't really optional

Reply:
Yes, they are compulsory, but I follow what the other files do, if you look in the data folder, almost everything is optional, I thought it was normal to mark them as optional
I will fix it for my next push